### PR TITLE
[React Aria Button] - Migrate sidebar, notifications, transactions, recurring schedule picker buttons

### PR DESCRIPTION
**Pull request by joel-jeremy**

### CODE REVIEW

File: packages/desktop-client/e2e/accounts.test.js-snapshots/Accounts-closes-an-account-4-chromium-linux.png
Comment (MatissJanis) [inline packages/desktop-client/e2e/accounts.test.js-snapshots/Accounts-closes-an-account-4-chromium-linux.png:1]: bug: the transfer icon color has changed. It's basically invisible now.

File: packages/desktop-client/e2e/budget.test.js-snapshots/Budget-renders-the-summary-information-available-funds-overspent-budgeted-and-for-next-month-1-chromium-linux.png
Comment (MatissJanis) [inline packages/desktop-client/e2e/budget.test.js-snapshots/Budget-renders-the-summary-information-available-funds-overspent-budgeted-and-for-next-month-1-chromium-linux.png:1]: bug: the notes icons are always visible now. Should only be visible on hover.

**Review by MatissJanis — CHANGES_REQUESTED** [review]

**Comment by MatissJanis** [comment]
Compared to `edge` - the notes button hover is still behaving quite differently than before. The new behaviour makes notes much less discoverable.

File: packages/desktop-client/e2e/accounts.test.js-snapshots/Accounts-closes-an-account-5-chromium-linux.png
Comment (matt-fidd) [inline packages/desktop-client/e2e/accounts.test.js-snapshots/Accounts-closes-an-account-5-chromium-linux.png:1]: The hover effect on the mealballs (?) menu in these screenshots is new. Is there an unintended focus change on load that only appears on VRT? Seems to work as expected in the demo deployment so I assume it's a symptom of the VRT env.

**Review by MikesGlitch — APPROVED** [review]
I think this looks great. I've given it a runthrough and haven't seen anything out of place.

**Review by MatissJanis — APPROVED** [review]

**Review by matt-fidd — APPROVED** [review]

**Pull request merged**